themes/prism.css

```diff
@@ -104,7 +104,7 @@ pre[class*="language-"] {
 .token.url,
 .language-css .token.string,
 .style .token.string {
-	color: #9a6e3a;
+	color: #a67f59;
```

Comment:
This change should not be here. This file was changed 10 days ago in https://github.com/PrismJS/prism/commit/8aea939a6287f0147d1d64d3f0a238b6cadbc9c6. `#9a6e3a` is the correct value.